src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is {person.name}
+    </h2>
+    {person.age
+      && (
+        <p className="Person__age">
+          I am {person.age}
+        </p>
+      )}
+    <p className="Person__partner">{person.isMarried ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'}</p>
```

Comment:
This line looks too complicated and hard-reading.
Please create a separate variable for partner text before to make your code cleaner